common/interfaces/quiz.ts

```diff
@@ -0,0 +1,5 @@
+export interface Quiz {
+    quizId: number;
+    maxDuration: number;
+    roomId: number;
```

Comment:
What happens if the quiz does not have a Room ID?

Reply:
Depends on the endpoint implementation. I haven't had the chance to code an endpoint that needed a quiz ID so far, but if I were to write one, it'd send a 400 BAD REQUEST answer to the client if it was missing arguments